public/apps/configuration/panels/user-list.tsx

```diff
@@ -0,0 +1,199 @@
+      setSelection([]);
+    } catch (e) {
+      console.log(e);
+    } finally {
+      setActionsPopoverOpen(false);
+    }
+  };
+
+  const actionsMenuItems = [
+    <EuiContextMenuItem
+      key="edit"
+      onClick={() => {}} // TODO: Redirect to edit page
+      disabled={selection.length !== 1}
+    >
+      Edit
+    </EuiContextMenuItem>,
+    // TODO: Redirect to duplicate page
+    <EuiContextMenuItem key="duplicate" onClick={() => {}} disabled={selection.length !== 1}>
+      Duplicate
+    </EuiContextMenuItem>,
+    <EuiContextMenuItem
+      key="delete"
+      onClick={handleDelete}
+      disabled={selection.length === 0 || selection.some((e) => e.username === currentUsername)}
+    >
+      Delete
+    </EuiContextMenuItem>,
+  ];
+
+  const actionsButton = (
+    <EuiButton
+      iconType="arrowDown"
+      iconSide="right"
+      onClick={() => {
+        setActionsPopoverOpen(true);
+      }}
+    >
+      Actions
+    </EuiButton>
+  );
+
+  return (
+    <>
+      <EuiPageHeader>
+        <EuiTitle size="l">
+          <h1>Internal users</h1>
+        </EuiTitle>
+      </EuiPageHeader>
+      <EuiPageContent>
+        <EuiPageContentHeader>
+          <EuiPageContentHeaderSection>
+            <EuiTitle size="s">
+              <h3>Internal users ({userData.length})</h3>
+            </EuiTitle>
+            <EuiText size="xs" color="subdued">
+              The Security plugin includes an internal user database. Use this database in place of
+              or in addition to an external authentication system such as LDAP or Active Directory.
+              You can map an internal user to a role from Roles. First, click into the detail page
+              of the role. Then under “Mapped users”, click “Manage mapping”.{' '}
+              <EuiLink external={true} href="/">
+                Learn More
+              </EuiLink>
+            </EuiText>
+          </EuiPageContentHeaderSection>
+          <EuiPageContentHeaderSection>
+            <EuiFlexGroup>
+              <EuiFlexItem>
+                <EuiPopover
+                  id="actionsMenu"
+                  button={actionsButton}
+                  isOpen={isActionsPopoverOpen}
+                  closePopover={() => {
+                    setActionsPopoverOpen(false);
+                  }}
+                  panelPaddingSize="s"
+                >
+                  <EuiContextMenuPanel items={actionsMenuItems} />
+                </EuiPopover>
+              </EuiFlexItem>
+              <EuiFlexItem>
+                <EuiButton fill>Create internal user</EuiButton>
+              </EuiFlexItem>
+            </EuiFlexGroup>
+          </EuiPageContentHeaderSection>
+        </EuiPageContentHeader>
+        <EuiPageBody>
+          <EuiInMemoryTable
+            loading={userData === [] && !errorFlag}
+            columns={columns}
+            items={userData}
+            itemId={'username'}
+            pagination
+            search
```

Comment:
There is a placeholder in the search bar which you can add by the following code

`
  const search = {
    box: {
      placeholder: 'Search internal users',
    },
  };
`

Reply:
Didn't notice, updated.